collection-scripts/gather_network_logs

```diff
@@ -23,6 +23,24 @@ for NODE in $NODES; do
     PIDS+=($!)
     oc -n openshift-sdn exec $OVS_POD -- bash -c "cat /var/log/openvswitch/ovsdb-server.log" > ${NETWORK_LOG_PATH}/${OVS_POD}_ovsdb_log &
     PIDS+=($!)
+
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "ip a" > ${NETWORK_LOG_PATH}/${NODE}_ip &
+    PIDS+=($!)
+
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "brctl show" > ${NETWORK_LOG_PATH}/${NODE}_brctl &
+    PIDS+=($!)
+
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "bridge fdb show" >> ${NETWORK_LOG_PATH}/${NODE}_bridgefdb &
+    PIDS+=($!)
+
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "bridge vlan show" >> ${NETWORK_LOG_PATH}/${NODE}_bridgevlan &
+    PIDS+=($!)
+
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "iptables -t filter -L" >> ${NETWORK_LOG_PATH}/${NODE}_iptables_filter &
+    PIDS+=($!)
+
+    oc -n openshift-sdn exec $SDN_POD -- bash -c "iptables -t nat -L" >> ${NETWORK_LOG_PATH}/${NODE}_iptables_nat &
+    PIDS+=($!)
```

Comment:
Can these two commands be condensed? 

Something like `iptablles -c`

Reply:
I'm not aware of a way to condense it. Skipping the table  (-t) just defaults to listing only 'filter'.
Actually for completeness we should have:
iptables -L -t filter
iptables -L -t nat
iptables -L -t mangle
iptables -L -t raw
iptables -L -t security

All of this is available in "iptables-save -c" we do above, but "iptables" is a lot nicer to read.

Reply:
The `iptables-save -c` output is complete and unambiguous, while the `iptables -L` output is neither. We shouldn't add a gratuitous second copy of the iptables data, which may be quite large on large clusters.